sass/components/navbar.sass

```diff
@@ -124,9 +124,11 @@ $navbar-breakpoint: $desktop !default
 
 html,
 body
-  &.has-navbar-fixed-top
+  &.has-navbar-fixed-top,
+  &.has-spaced-navbar-fixed-top
     padding-top: $navbar-height
```

Comment:
I just saw this and I thought if `$navbar-padding-vertical * 2` should be added? Thoughts?

Reply:
To explain the suggestion above: when a navbar is fixed on top and it is spaced, content digs under because it won't consider the navbar's padding, regardless of whether it exists or not.

Does this make sense? Can I help?

Reply:
Already done on lines 422 and 424 in this file below.

`is-spaced` only adds extra padding from `$navbar-breakpoint` which defaults to `$desktop` by default. So, the changes here add the regular padding for viewports under that for `has-spaced-navbar-fixed-(top|bottom)` class.